profiling/proto/service/service.proto

```diff
@@ -0,0 +1,91 @@
+// Copyright 2019 gRPC authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// This package defines the proto messages and RPC services exposed by gRPC for
+// profiling management. A reference client implementation to interact with
+// this service is provided as a command-line application. This service can be
+// used to toggle profiling on and off and retrieve stats from a gRPC
+// application.
+option go_package = "google.golang.org/grpc/profiling/proto/service";
+
+// EnableRequest defines the fields in a /Profiling/Enable method request to
+// toggle profiling on and off within a gRPC program.
+message EnableRequest {
+  // Setting this to true will enable profiling. Setting this to false will
+  // disable profiling.
+  bool enabled = 1;
+}
+
+// EnableResponse defines the fields in a /Profiling/Enable method response.
+message EnableResponse {
+}
+
+// GetStreamStats defines the fields in a /Profiling/GetStreamStats method
+// request to retrieve stream-level stats in a gRPC client/server.
+message GetStreamStatsRequest {
+}
+
+// A TimerProto measures the start and end of execution of a component within
```

Comment:
s/Proto//

Reply:
I wanted to clarify that this is different from the `Timer` struct defined in the internal package. Or did you mean to suggest I change just the comment to refer to `Timer` whilst keeping the message name to be `TimerProto`?

As for why there are two different structs: https://github.com/grpc/grpc-go/pull/3158#discussion_r346082630

Reply:
I think it's fine to use the package name to distinguish the two types.  `pb.Timer` vs. `profiling.TImer`.  If you have both in one place you can name local variables `timerProto` and `timer` (or whatever else SGTY) to distinguish them.